bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/action/RoomActions.java

```diff
@@ -105,7 +97,8 @@ public void setThermRoomModeSetpoint(
         try {
             targetMode = SetpointMode.valueOf(mode);
             if (!ALLOWED_MODES.contains(targetMode)) {
-                logger.info("Mode can only be MAX, HOME or MANUAL for a room");
+                logger.info("Mode can only be {} for a room",
+                        ALLOWED_MODES.stream().map(s -> s.name()).collect(Collectors.joining(" or ")));
```

Comment:
Perhaps ", " would be a better separator, i.e. "MAX, HOME, MANUAL" instead of "MAX or HOME or MANUAL"?

Reply:
I hesitated. But ok to change.